docs/pages/guides/authentication.mdx

```diff
@@ -23,6 +23,7 @@ If you'd like to see more, you can [open a PR](https://github.com/expo/expo/edit
 
 <Grid>
   <GridItem title="IdentityServer 4" protocol={['OAuth 2', 'OpenID']} image={ASSETS.id4} />
+  <GridItem title="Keycloak" protocol={['OAuth 2', 'OpenID']} image={ASSETS.keycloak} />
```

Comment:
there is an image missing, can anyone add that please?

Reply:
Here you go: https://github.com/donni106/expo/pull/1